sdks/csharp/sdk/AgonesSDK.cs

```diff
@@ -11,60 +11,56 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+using Agones.Dev.Sdk;
+using Grpc.Core;
+using Microsoft.Extensions.Logging;
 using System;
 using System.Threading;
 using System.Threading.Tasks;
-using Grpc.Core;
-using Agones.Dev.Sdk;
 
 [assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Agones.Test")]
 namespace Agones
 {
-	public sealed class AgonesSDK : IDisposable
+	public sealed class AgonesSDK : IAgonesSDK
 	{
 		public string Host { get; } = Environment.GetEnvironmentVariable("AGONES_SDK_GRPC_HOST") ?? "localhost";
-		public int Port { get; } = Convert.ToInt32((Environment.GetEnvironmentVariable("AGONES_SDK_GRPC_PORT") ?? "9357"));
+		public int Port { get; } = Convert.ToInt32(Environment.GetEnvironmentVariable("AGONES_SDK_GRPC_PORT") ?? "9357");
 
 		/// <summary>
 		/// The timeout for gRPC calls.
 		/// </summary>
 		public double RequestTimeout { get; set; }

```

Comment:
Sure thing.  Is this sufficient?

```suggestion
		public double RequestTimeoutSec { get; set; }
```

Reply:
Looks great! Thanks!